internal/buildscripts/packaging/fpm/common.sh

```diff
@@ -38,7 +38,9 @@ install_pkg() {
     local pkg_base=$( basename "$pkg_path" )
 
     echo "Installing $pkg_base ..."
+    echo "$(ls -l $pkg_path)"
     docker cp "$pkg_path" $image_name:/tmp/$pkg_base
+    sleep 5
```

Comment:
Is there any way to actually wait for whatever we want to complete instead of the sleep? This may not fix the problem, but only make it less flaky.

Reply:
Yeah that's true. I'll take another look at this, thanks! 😄